conf/insecure_certificate.pem

```diff
@@ -15,31 +15,3 @@ IQHEwDe+XTguJBmdOSp15FCf7plggKMTmkgu2GRToZv/rCldJbLTgfC+QdE2zZ/O
 2AF271LMJiQm+DBIApSnHVwLUUm42f8wkPBKJcselAcybf1Iz8YVsL3KROeUsemz
 EbluOqHoallQcPeq9RoXnYZOUGjh6p9pLZ3nnNDA
 -----END CERTIFICATE-----
------BEGIN PRIVATE KEY-----
-MIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQDW0gIYobL7FafQ
```

Comment:
let's move the key back into the same file.  I'd probably prefer them in separate files too, but separating them has only been supported since haproxy 2.2 which dropped in June (https://www.haproxy.com/blog/announcing-haproxy-2-2/ ).  Lots of people will be running caldera on slower supported linux distros where this version hasn't (or won't) make it into the default package repositories.